cmd/fossa/main.go

```diff
@@ -64,7 +66,8 @@ func Run(ctx *cli.Context) error {
 	}
 
 	if config.APIKey() == "" && !ctx.Bool(analyze.ShowOutput) {
-		return cli.ShowAppHelp(ctx)
+		fmt.Printf("Incorrect Usage. FOSSA_API_KEY must be set as an environment variable or provided in .fossa.yml\n\n")
```

Comment:
[nit] Let's actually `log.Fatal` for now since error handling at the top level is kind of weird.